jupyterhub/spawner.py

```diff
@@ -515,6 +515,15 @@ def my_hook(spawner):
         """
     ).tag(config=True)
 
+    post_spawn_hook = Any(
```

Comment:
Can we call it `post_stop_hook`? I would expect `post_spawn` to be after start finishes (we probably should have called `pre_spawn` `pre_start`).